etsin_finder/qvain_light_resources.py

```diff
@@ -0,0 +1,120 @@
+# This file is part of the Etsin service
+#
+# Copyright 2017-2018 Ministry of Education and Culture, Finland
+#
+# :author: CSC - IT Center for Science Ltd., Espoo Finland <servicedesk@csc.fi>
+# :license: MIT
+
+"""RESTful API endpoints, meant to be used by Qvain Light form"""
+
+from functools import wraps
+
+from flask import request, session
+from flask_mail import Message
+from flask_restful import abort, reqparse, Resource
+
+from etsin_finder.app_config import get_app_config
+from etsin_finder import authentication
+from etsin_finder import authorization
+from etsin_finder import cr_service
+from etsin_finder import qvain_light_file_service
+from etsin_finder.finder import app
+from etsin_finder.utils import \
+    sort_array_of_obj_by_key, \
+    slice_array_on_limit
+
+log = app.logger
+
+TOTAL_ITEM_LIMIT = 1000
+
+
```

Comment:
Qvain Light specific file API resources. Makes potential separation of Qvain Light easier.